.github/validation_samples/inclusive/config.py

```diff
@@ -20,8 +20,8 @@
 p.run = int(os.environ['LDMX_RUN_NUMBER'])
 p.maxEvents = int(os.environ['LDMX_NUM_EVENTS'])
 
-p.histogramFile = 'hist.root'
-p.outputFiles = ['events.root']
+p.histogramFile = 'hist_8gev.root'
+p.outputFiles = ['events_8gev.root']
```

Comment:
The output histogram and event files should keep the same names as before so that they are picked up by the CI.

```suggestion
p.histogramFile = 'hist.root'
p.outputFiles = ['events.root']
```

Reply:
I changed the output file name, because probably the validation tool takes a key_val pair as the file name. I have again changed it to default one.

Reply:
That makes sense, the CI script pre-dates the Validation tool and we are hoping to unify them one day.